diff --git a/research/object_detection/exporter.py b/research/object_detection/exporter.py
index ef8fe194..95469e68 100644
--- a/research/object_detection/exporter.py
+++ b/research/object_detection/exporter.py
@@ -68,8 +68,7 @@ def freeze_graph_with_def_protos(
 
     if optimize_graph:
       logging.info('Graph Rewriter optimizations enabled')
-      rewrite_options = rewriter_config_pb2.RewriterConfig(
-          layout_optimizer=rewriter_config_pb2.RewriterConfig.ON)
+      rewrite_options = rewriter_config_pb2.RewriterConfig()
       rewrite_options.optimizers.append('pruning')
       rewrite_options.optimizers.append('constfold')
       rewrite_options.optimizers.append('layout')
